app.rb

```diff
@@ -0,0 +1,192 @@
+require_relative './book'
+require_relative './person'
+require_relative './student'
+require_relative './rental'
+require_relative './teacher'
+
+# App Class that will serve as your console app entry-point.
+
+@books_arr = []
+@people_arr = []
+@rentals_arr = []
+
+def book_store(book)
+  @books_arr.push(book)
+  @books_arr
+end
+
+def people_store(person)
+  @people_arr.push(person)
+  @people_arr
+end
+
+def rentals_store(rental)
+  @rentals_arr.push(rental)
+  @rentals_arr
+end
+
+def success(message)
+  puts "Successfully created #{message}"
+end
+
+def user_info
+  print 'Enter the name of the person: '
+  name = gets.chomp
+  print 'Age: '
+  age = gets.chomp
+  [name, age]
+end
+
+def user_input
+  puts 'Welcome to School Library App'
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+def list_books
+  @books_arr.each do |book|
+    puts "Title: #{book.title}, Author: #{book.author}"
+  end
+  main
+end
+
+def list_people
+  @people_arr.each { |person| puts "[#{person.class}]: Name: #{person.name}, ID: #{person.id} AGE: #{person.age}" }
+  main
+end
+
+def choice_permission
+  print 'Has parent permission? [Y/N]: '
+  parent_permission = gets.chomp
+  print 'Enter the classroom: '
```

Comment:
- [ ] <b>REQUIRED:</b> As mentioned in the previous review, classroom is not required in creating people, student or teacher. Kindly remove it.
![scrnli_5_14_2022_4-08-49 PM](https://user-images.githubusercontent.com/45217364/168429500-4b2ce1fe-8cc0-4678-826b-94e702f072d1.png)

Reply:
This has been resolved 👍